### PR TITLE
fix: Subscriber property

### PR DESCRIPTION
**Pull request by nick2432**
### What change does this PR introduce?

Basically, I updated the POST request code and sent firstName, lastName, and email in the request.

### Why was this change needed?

Closes https://github.com/novuhq/novu/issues/4096

### Other information (Screenshots)
before
Screenshot from 2023-11-13 09-57-43

after
Screenshot from 2023-11-12 12-30-57

### CODE REVIEW

File: apps/widget/src/components/notification-center/NotificationCenterWidget.tsx [hunk 1]
```diff
@@ -95,10 +99,6 @@ export function NotificationCenterWidget(props: INotificationCenterWidgetProps)
           setPreferenceFilter(() => data.value.preferenceFilter);
         }
 
-        if (data.value.showUserPreferences) {
-          setShowUserPreferences(data.value.showUserPreferences);
-        }
```
Comment (scopsy) [inline apps/widget/src/components/notification-center/NotificationCenterWidget.tsx:100]: Why was this removed here?

Comment (nick2432) [reply apps/widget/src/components/notification-center/NotificationCenterWidget.tsx]: yes 
Looks like I made a mistake.

**Comment by jainpawan21** [comment]
@nick2432 
I think we should add other properties like phone, locale and data as well

**Comment by jainpawan21** [comment]
Implementation seems to be correct

**Comment by nick2432** [comment]
> Implementation seems to be correct

ok i will add other properties

File: apps/widget/src/components/notification-center/NotificationCenterWidget.tsx [hunk 2]
```diff
@@ -134,6 +142,12 @@ export function NotificationCenterWidget(props: INotificationCenterWidgetProps)
           socketUrl={socketUrl}
           applicationIdentifier={props.applicationIdentifier as string}
           subscriberId={userDataPayload.subscriberId}
+          firstName={userDataPayload.firstName}
+          lastName={userDataPayload.lastName}
+          email={userDataPayload.email}
+          phone={userDataPayload.phone}
+          locale={userDataPayload.locale}
+          data={userDataPayload.data}
```
[the remark this answers is not in the document]
Comment (nick2432) [reply apps/widget/src/components/notification-center/NotificationCenterWidget.tsx:148]: Hey @Paweł T., is this correct?
Screenshot from 2023-12-14 09-34-03
Screenshot from 2023-12-14 09-34-20
